### PR TITLE
Bump modules `kms-key` to v0.11.0 and `sns-topic` to v0.20.1

### PR DESCRIPTION
**Pull request by milldr**
## what
* Update the modules versions for kms-key and sns-topic

## why
* Required to resolve downstream errors with the `compliance` components

## references
* closes #29
* closes #28

### CODE REVIEW

**Review by nitrocode — DISMISSED** [review]

File: .github/auto-release.yml [hunk 1]
```diff
@@ -46,7 +47,7 @@ template: |
 
 replacers:
 # Remove irrelevant information from Renovate bot
-- search: '/---\s+^#.*Renovate configuration(?:.|\n)*?This PR has been generated .*/gm'
+- search: '/(?<=---\s+)+^#.*(Renovate configuration|Configuration)(?:.|\n)*?This PR has been generated .*/gm'
```
Comment (Nuru) [inline .github/auto-release.yml]: This was supposed to be (will soon be) fixed in build-harness. For now, fix it manually.
```suggestion
- search: '/(?<=---\s)\s*^#.*(Renovate configuration|Configuration)(?:.|\n)*?This PR has been generated .*/gm'
```

**Review by Nuru — DISMISSED** [review]

**Review by Nuru — APPROVED** [review]

**Pull request merged**